website/docs/addon_usd_resolver_admin.md

```diff
@@ -0,0 +1,19 @@
+---
+id: addon_usd_resolver_admin
+title: AYON USD Resolver
+sidebar_label: USD Resolver
+description: AYON USD Resolver documentations for admins.
+toc_max_heading_level: 5
+---
+
+import ReactMarkdown from "react-markdown";
+import versions from '@site/docs/assets/json/Ayon_addons_version.json'
+
+<ReactMarkdown>
+  {versions.USD_Badge}
+</ReactMarkdown>
+
```

Comment:
Add details about what settings to enable to correctly use the resolver.

1. Require ayon-usd addon
2. Configure LakeFS settings
    - Explain app aliases
3. Enable Ayon URI load/generation settings

```
# houdini
ayon+settings://houdini/load/LOPLoadAssetLoader/use_ayon_entity_uri
ayon+settings://houdini/load/LOPLoadShotLoader/use_ayon_entity_uri
ayon+settings://houdini/load/USDSublayerLoader/use_ayon_entity_uri
ayon+settings://houdini/load/USDReferenceLoader/use_ayon_entity_uri
ayon+settings://houdini/load/SopUsdImportLoader/use_ayon_entity_uri

# maya
ayon+settings://maya/load/MayaUsdLoader/use_ayon_entity_uri

# core
ayon+settings://core/publish/ExtractUSDAssetContribution/use_ayon_entity_uri
ayon+settings://core/publish/ExtractUSDLayerContribution/use_ayon_entity_uri
```